src/main.cpp

```diff
@@ -351,6 +353,8 @@ int main(int argc, char **argv) {
             world.updateGameObjects(deltaTime, totalTime);
             camera.update(deltaTime);
 
+            gameManager.decreaseTime(deltaTime);

```

Comment:
Yeah, I didn't add a check if the time runs out, as the only option by now would be a exit call, but I wanted us to be able to have the game running unlimited for testing